docs/source/user_guide/calculations.rst

```diff
@@ -18,24 +27,82 @@ Below is a usage example with the minimum required parameters. These parameters
 
 .. code-block:: python
 
+    SinglePointCalculation = CalculationFactory("janus.sp")
+    submit(SinglePointCalculation, code=InstalledCode, structure=StructureData, metadata={"options": {"resources": {"num_machines": 1}}})
+
+The inputs can be grouped into a dictionary:
+
+.. code-block:: python
 
+    inputs = {
+            "metadata": {"options": {"resources": {"num_machines": 1}}},
+            "code": InstalledCode,
+            "architecture": Str,
+            "structure": StructureData,
+            "model": ModelData,
+            "precision": Str,
+            "device": Str,
+        }
     SinglePointCalculation = CalculationFactory("janus.sp")
-    submit(SinglePointCalculation, code=InstalledCode, structure=StructureData)
+    submit(SinglePointCalculation, **inputs)
+
+
+Or they can be passed as a config file. The config file has to be structured as it would be for a janus calculation (refer to `janus documentation <https://stfc.github.io/janus-core/apidoc/janus_core.html>`_ ) and passed as an AiiDA data type itself.
```

Comment:
```suggestion
Or they can be passed as a config file. The config file has to be structured as it would be for a janus calculation (refer to `janus documentation <https://stfc.github.io/janus-core/>`_ ) and passed as an AiiDA data type itself.
```
Optional but I'm thinking the main page might be better to link, since the API docs aren't the most user friendly starting point

Reply:
yeah I know, I put api cause at the moment the stuff is mostly there